cl/users/templates/profile/recap_email.html

```diff
@@ -0,0 +1,61 @@
+{% extends "profile/nav.html" %}
+{% load static %}
+
+{% block title %}@recap.email – CourtListener.com{% endblock %}
+
+{% block nav-account %}active{% endblock %}
+{% block footer-scripts %}
+  <script type="text/javascript"
+          src="{% static "js/jquery.bootstrap-growl.min.js" %}"></script>
+  <script src="{% static "js/htmx.min.js" %}"></script>
+  <script defer
+          type="text/javascript" src="{% static "js/toggle_auto_subscribe.js" %}"></script>
+  <link href="{% static "css/bootstrap4-toggle.min.css" %}" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+  {# Navigation Tabs #}
+  {% include "includes/account-tabs.html" %}
+
+  <div class="col-sm-2"></div>
+  <div class="col-xs-12 col-sm-8">
+    <p class="lead"><strong>@recap.email</strong> is a system we use to gather content from PACER and add it to the <a href="{% url 'advanced_r' %}">RECAP Archive</a>. If you receive notification emails from PACER, it only takes a minute to set up this system and begin contributing content to the public commons.
+    </p>
+
+    <h2>How it works</h2>
+    <p>Three easy steps:</p>
+    <ol>
+      <li>
+        <p>Every party in a case gets notification emails that contain one-time "magic links." These links give free access to new filings in the case.</p>
+      </li>
+      <li>
+        <p>We give you an email address that you can configure in your PACER account as a secondary recipient for these notifications. Your primary address in PACER is never changed.</p>
+        <p class="alert-warning alert">
+          {% if request.user.is_authenticated %}
+            Your @recap.email email address is <b>{{ request.user.profile.recap_email }}</b>
+          {% else %}
+            Log in and refresh this page to see your personal @recap.email address here.
+          {% endif %}
+        <p>
+      </li>
+      <li>After you add us as a secondary recipient in PACER, we'll get a copy of your notification emails. When we do, we'll add the items to RECAP, and we'll send you a notification so you're up to date.</li>
```

Comment:
This needs to be updated to mention auto-subscribe. Maybe:

```suggestion
      <li>
        <p>After you add us as a secondary recipient in PACER, we'll get a copy of your notification emails. When we do, we'll automatically add the items to RECAP.
        </p>
        <p>If you have <a href="{% url "view_recap_email" %}">auto-subscribe turned on in your profile</a> (the default), we will automatically start sending you email notifications for the case. If auto-subscribe is off, we will send you one email for each case you're associated with in PACER, when we get the first notification for that case from PACER. In the email we send, you can click a link to subscribe to it on CourtListener so you get all of its notifications, or if you do nothing, you won't get more emails for that case.
        </p>
      </li> 
```

Reply:
Thanks, added it into the include.